src/test/resources/alma-fix/990011470300206441.xml

```diff
@@ -56,25 +56,28 @@
     <subfield code="a">Politische Wissenschaft</subfield>
     <subfield code="D">s</subfield>
     <subfield code="0">(DE-588)4076229-4</subfield>
-    <subfield code="0">(uri) https://portal.dnb.de/opac.htm?method=simpleSearch&amp;cqlMode=true&amp;query=idn=040762297</subfield>
-    <subfield code="0">(uri) http://viaf.org/viaf/sourceID/DNB|040762297</subfield>
+    <subfield code="0">https://portal.dnb.de/opac.htm?method=simpleSearch&amp;cqlMode=true&amp;query=idn=040762297</subfield>
+    <subfield code="0">http://viaf.org/viaf/sourceID/DNB|040762297</subfield>
+    <subfield code="B">GND-040762297</subfield>
   </datafield>
   <datafield tag="700" ind1="1" ind2=" ">
     <subfield code="a">Nelson, John S.</subfield>
     <subfield code="d">1950-</subfield>
     <subfield code="0">(DE-588)172283159</subfield>
     <subfield code="4">edt</subfield>
-    <subfield code="0">(uri) https://portal.dnb.de/opac.htm?method=simpleSearch&amp;cqlMode=true&amp;query=idn=172283159</subfield>
-    <subfield code="0">(uri) http://viaf.org/viaf/sourceID/DNB|172283159</subfield>
+    <subfield code="0">https://portal.dnb.de/opac.htm?method=simpleSearch&amp;cqlMode=true&amp;query=idn=172283159</subfield>
```

Comment:
(uri) seems to be dropped, this has has the consequence that altLabels are not mapped above.